crates/bevy_ecs/src/lib.rs

```diff
@@ -701,87 +703,6 @@ mod tests {
         assert_eq!(added, vec![e2]);
     }
 
-    #[test]
-    fn mutated_trackers() {
```

Comment:
This test should be adapted to test Changed instead rather than removing it completely.

Reply:
Done in 77139839bc022b55f3bec5d291ab8c3322dd86e7.